core/src/main/java/io/stargate/core/metrics/impl/MetricsImpl.java

```diff
@@ -20,4 +42,14 @@ public MetricRegistry getRegistry() {
   public MetricRegistry getRegistry(String prefix) {
     return new PrefixingMetricRegistry(registry, prefix);
   }
+
+  @Override
+  public MeterRegistry getMeterRegistry() {
+    return prometheusMeterRegistry;
+  }
+
+  @Override
+  public String scrape() {
```

Comment:
is your goal to have info here about the `tenant_id`?

Reply:
So the tags can only be added to Micrometer, so yes I intent to extend the functionality to support tags and make it generic.. The `scrape()` itself delivers both Dropwizard and Micrometer metrics..